content/docs/gke/private-clusters.md

```diff
@@ -134,39 +134,34 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
 1. Create the access level
 
     ```
-    gcloud beta access-context-manager levels create cloudbuild \
+    gcloud beta access-context-manager levels create kubeflow \
        --basic-level-spec=members.yaml \
        --policy=${POLICYID} \
-       --title="Cloud Build"
+       --title="Kubflow ${PROJECT}"
     ```
 
+     * The name for the level can't have any hyphens
+
 1. Bind Access Level to a Service Perimeter
 
     ```
     gcloud beta access-context-manager perimeters update KubeflowZone \
-     --add-access-levels=cloudbuild \
+     --add-access-levels=kubeflow \
      --policy=${POLICYID}
     ```
-
-    * gcloud will return an error even though the job is
-      submited successfully and will run successfully
-      see [kubeflow/kubeflow#3105](https://github.com/kubeflow/kubeflow/issues/3105)
-
-    * You can use the Cloud console to monitor your GCB job.
-
 1. Setup container registry for GKE private clusters (for more info see [instructions](https://cloud.google.com/vpc-service-controls/docs/set-up-gke))
 
     1. Create a managed private zone
 
         ```
-        export ZONE_NAME=kubeflow
-        export NETWORK=<Network you are using for your cluster>
-        gcloud beta dns managed-zones create ${ZONE_NAME} \
-         --visibility=private \
-         --networks=https://www.googleapis.com/compute/v1/projects/${PROJECT}/global/networks/${NETWORK} \
-         --description="Kubeflow DNS" \
-         --dns-name=gcr.io \
-         --project=${PROJECT}
+          export ZONE_NAME=kubeflow
```

Comment:
There seems to be too much space in front of these code lines in the rendered preview, now that the indentation is increased.